flank-scripts/build.gradle.kts

```diff
@@ -28,7 +28,7 @@ shadowJar.apply {
     }
 }
 // <breaking change>.<feature added>.<fix/minor change>
-version = "1.3.2"
+version = "1.3.3"
```

Comment:
hmmm check if this is not the same on develop

Reply:
Yup, it wasn't but the master is changing all the time, will update it

Reply:
Hm...master is still having 1.3.2 (for now at least)